src/pages/components/main.js

```diff
@@ -1,5 +1,6 @@
 import { LayoutProviderService } from '../../services/layout-provider.service';
 
+import '../../assets/scss/buttons.scss';
```

Comment:
This import should be add to components/style.scss

Reply:
It was already there o.o

Reply:
The import is on the js file but need to be placed on the scss file.

Reply:
Done